attom2/between-the-lions
# Feature/inventory

#### What's this PR do?

Sets up fetch calls to get books from Google Books API
Adds component to view the inventory of books
Adds button the when clicked, routes to new page with embedded book

#### How should this be manually tested?

Open the app and see that books load, Click on a button to see new page is embedded

#### Any background context you want to provide?

#### Screenshots (if appropriate)
Sorry for your eyes, needs some subtle styling
![image](https://user-images.githubusercontent.com/49999607/89050780-7ab76c00-d310-11ea-9af2-7b009c0e7e03.png)
![image](https://user-images.githubusercontent.com/49999607/89050903-a6d2ed00-d310-11ea-97bf-1d9ae91d8d8a.png)

#### Questions:
